lib/trln_argon/view_helpers/trln_argon_helper.rb

```diff
@@ -67,6 +67,16 @@ def consortium_long_name
         t('trln_argon.consortium.long_name')
       end
 
+      def holdings_list
+        list = TrlnArgon::Engine.configuration.sort_order_in_holding_list
+        list.gsub('unc', 'UNC')
```

Comment:
There are translations available in the app that are designed to map the institution code to a display value. It would be better to use these existing translations.

Reply:
https://github.com/trln/trln_argon/blob/master/config/locales/trln_argon.en.yml#L58